presentation/components/src/presentation-components/common/Utils.ts

```diff
@@ -55,6 +57,26 @@ export const initializeLocalization = async () => {
   return () => Presentation.i18n.unregisterNamespace(localizationNamespaceName);
 };
 
+/**
+ * Registers custom property value renderers and returns cleanup callback that unregisters them.
+ * @internal
+ */
+export const initializePropertyValueRenderers = async () => {
+  const customRenderers: Array<{ name: string, renderer: IPropertyValueRenderer }> = [
+    { name: "selectable_instance", renderer: new InstanceKeyValueRenderer() },
```

Comment:
The idea is that consumers will specify this renderer name in their rulesets to get specific properties rendered with this renderer. I think we need two things.
- Maybe it's subjective, but I think "SelectableInstance" name would be more consumer-friendly than "selectable_instance", especially since we already use PascalCase in many other places.
- Need a documentation page that lists all custom renderers that we deliver. The `PropertyRendererSpecification` docs should reference that page to make those custom renderers a bit easier to find.

Reply:
- This is definitely a taste question, I don't mind either way.
- The documentation is the last step. It will bring together all these property rendering enhancement changes.